src/main/java/uk/gov/companieshouse/overseasentitiesapi/utils/HashHelper.java

```diff
@@ -0,0 +1,28 @@
+package uk.gov.companieshouse.overseasentitiesapi.utils;
```

Comment:
This is now merged so can update PR.

Reply:
Unfortunately because that branch was squashed there doesn't seem to be a way to unpick the previous commits from the one in this branch 

I've made a new branch and will be doing these comments in there tomorrow